index.js

```diff
@@ -1,35 +1,41 @@
 // Do not process anything if the env is develop
-if (process.env.ENVIRONMENT && process.env.ENVIRONMENT === 'develop') {
-  return
+
+const trueValues = [true, 'true', 'TRUE'];
+
+if (
+  (process.env.VERBOSE && trueValues.includes(process.env.VERBOSE))
+  || (process.env.WTF && trueValues.includes(process.env.WTF))
```

Comment:
I know one meaning of WTF :) but what does the WTF abbreviation stand for in this context?

Reply:
Exactly the meaning you know ;)

Reply:
Then no :)  could you remove that please